catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -159,24 +159,28 @@ private Map<String, String> appendNecessaryProperties(Map<String, String> proper
     // If the backend server is less than DEFAULT_REPLICATION_FACTOR_IN_SERVER_SIDE (3), we need to
     // set the property 'replication_num' to 1 explicitly.
     if (!resultMap.containsKey(REPLICATION_FACTOR)) {
-      // Try to check the number of backend servers.
-      String query = "select count(*) from information_schema.backends where Alive = 'true'";
+      // Try to check the number of backend servers using show backends command
+      String query = "show backends";
```

Comment:
I think the comment should explain "why", not "how".

Reply:
Thanks.  Updated